packages/astro/src/core/build/generate.ts

```diff
@@ -67,7 +67,8 @@ export function chunkIsPage(astroConfig: AstroConfig, output: OutputAsset | Outp
 }
 
 export async function generatePages(result: RollupOutput, opts: StaticBuildOptions, internals: BuildInternals, facadeIdToPageDataMap: Map<string, PageBuildData>) {
-	info(opts.logging, null, `\n${bgMagenta(black(' generating static routes '))}\n`);
+	const timer = performance.now();
+	info(opts.logging, null, `\n${bgGreen(black(' generating html '))}`);
```

Comment:
This isn't _always_ going to be HTML if you have JSON routes! May prefer the original wording for clarity

```suggestion
	info(opts.logging, null, `\n${bgGreen(black(' generating static routes '))}`);
```

Reply:
good call!